<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>661</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>661</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>课程管理助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>621</width>
      <height>421</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>621</width>
      <height>421</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actiond"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>添加</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>删除</string>
    </property>
    <addaction name="action_12"/>
    <addaction name="separator"/>
    <addaction name="action_14"/>
    <addaction name="separator"/>
    <addaction name="action_16"/>
    <addaction name="separator"/>
    <addaction name="action_18"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_36"/>
    <addaction name="separator"/>
    <addaction name="action_21"/>
    <addaction name="separator"/>
    <addaction name="action_23"/>
    <addaction name="separator"/>
    <addaction name="action_27"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>检索</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_20"/>
    <addaction name="separator"/>
    <addaction name="action_22"/>
    <addaction name="separator"/>
    <addaction name="action_25"/>
    <addaction name="separator"/>
    <addaction name="action_29"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>修改</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="separator"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="separator"/>
    <addaction name="action_19"/>
    <addaction name="separator"/>
    <addaction name="action_24"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiond">
   <property name="text">
    <string>存储</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>教师</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>教室</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>教师</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>教室</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>显示所有</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>课堂</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>课堂</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>教师</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>教室</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>课堂</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>教室</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="action_30">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action_32">
   <property name="text">
    <string>课堂</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>教师</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>教室</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>班级</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>课堂</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>空闲教室查找</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>教室利用与能效分析</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>教师承担课堂合理性分析</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>班级参与课堂的合理性分析</string>
   </property>
  </action>
  <action name="action_36">
   <property name="text">
    <string>空闲教室与检索</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
